src/browser/api_protocol/api_handlers/system.ts

```diff
@@ -71,6 +71,7 @@ export const SystemApiMap: APIHandlerMap = {
     'get-focused-window': getFocusedWindow,
     'get-focused-external-window': getFocusedExternalWindow,
     'get-host-specs': { apiFunc: getHostSpecs, apiPath: '.getHostSpecs' },
+    'get-installed-runtimes': getInstalledRuntimes,
```

Comment:
Can we style this one similar to `'get-host-specs': { apiFunc: getHostSpecs, apiPath: '.getHostSpecs' },` above it? The reason, while unfortunately not obvious, is that by doing so we can control it via group policy security settings.